packages/chat/tests/messageUtils.test.ts

```diff
@@ -0,0 +1,110 @@
+
+	test('Less than maxMsgLength chars should result in an array of length 1', () => {
+		expect(autoSplitterWithSpaces('This is a test.', 500)).toStrictEqual(['This is a test.']);
+	});
+
+	test('With a command and less than maxMsgLength chars should result in an array of length 1', () => {
+		expect(autoSplitterWithSpaces('/vip testChannel', 500)).toStrictEqual(['/vip testChannel']);
+	});
+
+	test('Character limit of 1', () => {
+		expect(autoSplitterWithSpaces('/vip testChannel', 1)).toStrictEqual([
+			'/',
+			'v',
+			'i',
+			'p',
+			't',
+			'e',
+			's',
+			't',
+			'C',
+			'h',
+			'a',
+			'n',
+			'n',
+			'e',
+			'l'
+		]);
+	});
+
+	test('Over character limit and should have 2 messages', () => {
+		expect(autoSplitterWithSpaces(testString.repeat(35), 500)).toStrictEqual([
+			'This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This',
+			'is a test This is a test'
+		]);
+	});
+
+	test('Over character limit and should have 3 messages', () => {
+		expect(autoSplitterWithSpaces(testString.repeat(70), 500)).toStrictEqual([
+			'This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This',
+			'is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a test This is a',
+			'test This is a test This is a test This is a test'
+		]);
+	});
+
+	test('Over character limit and should have 4 messages', () => {
```

Comment:
Not sure whether this test case is necessary. If we know we can go past just two parts, no need to test for another, IMO. Also, the expected result doesn't even have 4 messages.